### PR TITLE
Minor fixes to perfect forwarding section

### PR DESCRIPTION
**Pull request by eguiraud**
- Remove a wrong statement: rvalues would actually be supported if wrapper took a constT&
- Preserve value type in wrapper function

### CODE REVIEW

**Review by sponce — APPROVED** [review]

File: talk/expert/perfectforwarding.tex [hunk 1]
```diff
@@ -57,7 +57,6 @@
   \begin{alertblock}{}
     \begin{itemize}
       \item \mintinline{cpp}{const T&} won't work when passing something non const
-      \item rvalues are not supported in either case
```
Comment (bernhardmgruber) [inline talk/expert/perfectforwarding.tex:60]: Why is this removed? It should be at least reworded in: `rvalues are also not supported`. In the example `wrapper(42)`, `42` is an rvalue (a prvalue), and cannot be bound to a mutable reference. This is important here, because otherwise `wrapper(T& arg)` could handle all cases.

Comment (bernhardmgruber) [reply talk/expert/perfectforwarding.tex:60]: True. I would prefer to have that in the summary of the slide at the bottom though. But if you insist, we can leave it as is.
